modules/osdk-bundle-validate-tests.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-bundle-validate.adoc
+
+:_content-type: REFERENCE
+[id="osdk-bundle-validate-tests_{context}"]
+= Built-in bundle validate tests
+
+The Operator SDK ships with pre-defined `bundle validate` tests arranged into suites. If you run the `bundle validate` command without specifying a test, the default test runs. The default test verifies a bundle adheres to the specifications in the upstream link:https://github.com/operator-framework/operator-registry/blob/master/docs/design/operator-bundle.md[Operator Bundle Overview].
+
+You can run your bundle against additional tests, such as the requirements necessary to publish your Operator on the OperatorHub or to check for deprecated Kubernetes APIs. Any additional tests run along with the default test.
```

Comment:
The "Any additional tests run along with the default test." feels awkward like there should be more or rephrased.

Reply:
Maybe I'm parsing it wrong. So when I read it, I see the beginning of a sentence that says additional tests run along with the default test WILL DO SOMETHING ELSE. 

but I think what you are trying to say that "additional tests" always run in addition to the default test. 

If the latter is what you mean, then I think it is phrased correctly, I just parsed it wrong.

Reply:
I have tried to rephrase it to make it clearer. If you need to interpret and think abou what I am trying to say then a customer is likely to have a hard time too.